src/python-common/ceph/deployment/hostspec.py

```diff
@@ -1,9 +1,22 @@
+import errno
 try:
     from typing import Optional, List, Any
 except ImportError:
     pass  # just for type checking
 
 
+class HostSpecValidationError(Exception):
```

Comment:
```suggestion
class SpecValidationError(Exception):
```
and just use it everywhere? I doubt anyone needs to differentiate them.